### ISSUE TITLE
Compile time value and Runtime value of env conflict (blockscout-v4.1.8)

### ISSUE THREAD

**Issue by LuxesOwen** [blockscout/blockscout#6709]
ERROR! the application :block_scout_web has a different value set for key BlockScoutWeb.ApiRouter during runtime compared to compile time. Since this application environment entry was marked as compile time, this difference can lead to different behaviour than expected:

  * Compile time value was set to: [writing_enabled: true, reading_enabled: false, wobserver_enabled: false]
  * Runtime value was set to: [writing_enabled: true, reading_enabled: true, wobserver_enabled: false]
  
  Then I add env export VALIDATE_COMPILE_ENV="false",but still can't startup.

**Comment by LuxesOwen** [comment]
<img width="859" alt="image" src="https://user-images.githubusercontent.com/66360114/212040492-a92d67e7-6adf-4bab-930e-d9d4b45a6cb2.png">
This the screenshot of your docs.How to change the compile env variables?

**Comment by LuxesOwen** [comment]
I reinstall it and change env variables it gone well. I think it can't been modify for twice.

**Comment by vbaranov** [comment]
@LuxesOwen when changing compile-time env variables values you should recompile the application.

**Comment by LuxesOwen** [comment]
@vbaranov So I can't change the variables values through change Runtime value? In order to pass the check,I must change both Compile time and Runtime value to same values?

**Comment by vbaranov** [comment]
@LuxesOwen yes, however, actually, a very tiny amount of environment variables are compile-time variables: those variables are marked at "Need recompile" column in https://docs.blockscout.com/for-developers/information-and-settings/env-variables. I am clsoing this issue. Please, let me know, if you still have questions.
